Gemfile

```diff
@@ -1,18 +1,21 @@
-source 'https://rubygems.org'
+source "https://rubygems.org"
 
 gemspec
 
 group :development do
-  if RUBY_VERSION >= '2.0.0'
-    gem 'pry-byebug'
+  if RUBY_VERSION >= "2.0.0"
+    gem "pry-byebug"
   else
-    gem 'pry-debugger'
+    gem "pry-debugger"
   end
-  gem 'wwtd'
-  gem 'travis'
+
+  gem "benchmark-ips"
 end
 
 group :test do
-  gem 'rake',  '~> 10.1'
-  gem 'rspec', '~> 3.0.0'
+  gem "rake",     "~> 10.4"
```

Comment:
There's a newer version (12.0.0) If it's possible, why not update it to this one? :grin: